### PR TITLE
Video-57-Choose-Address-On-Google-Map

### PR DESCRIPTION
**Pull request by basir**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
My map is not working...Only the loading box is working...no map is being displayed..Any advice??

**Review by WaseemAijaz — COMMENTED** [review]
hello sir,
my map is not showing. its just loading & loading & just loading

**Comment by WaseemAijaz** [comment]
> My map is not working...Only the loading box is working...no map is being displayed..Any advice??

same here from my side. please help me out

File: frontend/src/screens/MapScreen.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            </div>
+          </StandaloneSearchBox>
+          <Marker position={location} onLoad={onMarkerLoad}></Marker>
+        </GoogleMap>
+      </LoadScript>
+    </div>
+  ) : (
+    <LoadingBox></LoadingBox>
+  );
+}
```
Comment (Royal6969) [inline frontend/src/screens/MapScreen.js:124]: In the second 25:28 you press the OK in window.confirm()... but when you go back to ShippingAddressScreen in the next second 25:29 you haven´t got the address fields filled ... you select manually the address from your Google Account ... 
Something like this happends at first in 0:22 when you are going to do the test and you already have the fields filled ...
It is ok ? or maybe what have to happend is, when you choose your address in GoogleMaps and you press OK in the window.confirm(), the fields would have to fill them automatically ... anyone more realized that and think it ??

**Pull request merged**